demo/assets.js

```diff
@@ -1097,6 +1141,46 @@ shakaAssets.testAssets = [
       shakaAssets.Feature.MP4,
       shakaAssets.Feature.SEGMENT_TEMPLATE_DURATION
     ]
+  },
+  {
+    name: 'Big Buck Bunny',
+    manifestUri: 'https://content.uplynk.com/224ac8717e714b68831997ab6cea4015.mpd', // gjslint: disable=110
+
+    source: shakaAssets.Source.UPLYNK,
+    drm: [
+      shakaAssets.KeySystem.WIDEVINE
+    ],
+    features: [
+      shakaAssets.Feature.MP4,
+      shakaAssets.Feature.SEGMENT_LIST_DURATION
+    ],
+    licenseServers: {
+      'com.widevine.alpha': 'https://content.uplynk.com/wv',
+      'org.w3.clearkey': 'https://content.uplynk.com/ck'
+    },
+    requestFilter: shakaAssets.UplynkRequestFilter,
+    responseFilter: shakaAssets.UplynkResponseFilter
+  },
+  {
+    name: 'Sintel - (multiperiod-mix of encrypted and unencrypted)',
+    //Unencrypted periods interspersed with protected periods
+    //Requires Chrome 58
+    manifestUri: 'https://content.uplynk.com/1eb40d8e64234f5c9879db7045c3d48c.mpd?ad=cleardash', // gjslint: disable=110
+
+    source: shakaAssets.Source.UPLYNK,
+    drm: [
+      shakaAssets.KeySystem.WIDEVINE
+    ],
+    features: [
+      shakaAssets.Feature.MP4,
+      shakaAssets.Feature.SEGMENT_LIST_DURATION
```

Comment:
Done -- I also added Feature.MULTILINGUAL as I was reviewing the features.  This asset has a second Portuguese audio track.   We could add a new feature for assets like this that have completely clear periods interspersed with encrypted periods too.  I believe you haven't had many tests like this up to this point because of Chrome's issues supporting it until 58.